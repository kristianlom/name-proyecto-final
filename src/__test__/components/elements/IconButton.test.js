// import React from 'react';
// import { mount } from 'enzyme';
// import IconButton from '../../../components/elements/IconButton';

// describe('Testing render <IconButton /> Component', () => {
//   let wrapperComponent;
//   beforeEach(() => {
//     wrapperComponent = mount(<IconButton />);
//   });
//   // ---Snapshot:
//   it('create snapshot', () => {
//     expect(wrapperComponent.find(IconButton)).toMatchSnapshot();
//   });
//   //  ---Rendering:
//   it('component renders correctly', () => {
//     expect(wrapperComponent.find(IconButton).length).toEqual(1);
//   });
// });
