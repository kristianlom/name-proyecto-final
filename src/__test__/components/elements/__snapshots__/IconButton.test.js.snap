// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing render <IconButton /> Component create snapshot 1`] = `ReactWrapper {}`;
